conda_smithy/ci_register.py

```diff
@@ -178,48 +178,42 @@ def add_project_to_travis(user, project):
     headers = travis_headers()
     endpoint = 'https://api.travis-ci.org'
 
-    url = '{}/hooks'.format(endpoint)
+    url = '{}/repos/{user}/{project}'.format(endpoint, user=user, project=project)
 
-    found = False
     count = 0
 
-    while not found:
+    while True:
         count += 1
 
         response = requests.get(url, headers=headers)
         try:
             response.raise_for_status()
             content = response.json()
-        except (requests.HTTPError, ValueError):
+            if "repo" in content:
+                break
+        except requests.HTTPError:
             # We regularly seem to hit this issue during automated feedstock registration on github.
             # https://github.com/conda-forge/conda-smithy/issues/233
             # ValueError: No JSON object could be decoded
             # Maybe trying again in a few seconds will fix this.
```

Comment:
Probably could update the last two lines of this comment to be more informative knowing what we do. Something along the lines of the repo is not yet available via Travis CI's API.